src/components/v2/V2Project/V2ManageTokensSection/V1ProjectTokensSection/MigrateV1ProjectTokensModal/MigrateV1ProjectTokensForm.tsx

```diff
@@ -0,0 +1,67 @@
+import { Trans } from '@lingui/macro'
```

Comment:
This form is:

**Tokens to swap**

<img width="557" alt="Screen Shot 2022-07-14 at 2 57 39 PM" src="https://user-images.githubusercontent.com/94939382/179124843-3d749f9c-356c-442a-9ba1-1381fa305644.png">